llama_index/prompts/prompts.py

```diff
@@ -131,3 +131,9 @@
 Required template variables: `query_str`, `df_str`, `instruction_str`.
 """
 PandasPrompt = Prompt
+
+"""JSON Path prompt. Convert query to JSONPath query.
+
+Required template variables: `query_str`.
+"""
+JSONPathPrompt = Prompt
```

Comment:
Probably not clear from the codebase, but we started deprecating these type specific prompt classes.